staging/src/k8s.io/apiserver/pkg/endpoints/openapi/openapi.go

```diff
@@ -146,6 +146,15 @@ func friendlyName(name string) string {
 	return strings.Join(nameParts, ".")
 }
 
```

Comment:
please reconsider the commit title, or split it up. This change is not only about adding the config.

Reply:
changed title